src/Common/src/System/Net/NegotiationInfoClass.cs

```diff
@@ -11,5 +11,6 @@ internal partial class NegotiationInfoClass
         internal const string NTLM = "NTLM";
         internal const string Kerberos = "Kerberos";
         internal const string Negotiate = "Negotiate";
+        internal const string Basic = "Basic";
```

Comment:
I see that this optimization was made to avoid having `AuthConstants.cs`.
* Does "Basic" make sense in the context of Negotiate? 
* Where is this value used?